client/src/pages/createratingsystem.tsx

```diff
@@ -0,0 +1,22 @@
+import { Heading, Container, Stack } from "@chakra-ui/react";
+import * as React from 'react';
+import CreateRatingSystemForm from "../components/ratingsystem/CreateRatingSystemForm";
+
+const CreateRatingSystem: React.FC<{}> = () => {
+
+  return (
+    <Stack
+      as={Container}
+      spacing={{ base: 8, md: 14 }}
+      py={{ base: 20, md: 20 }}
+      maxWidth="3xl"
+    >
+      <Heading>
```

Comment:
This page should only be shown if the user is logged in. A check using meQuery can do and redirect the user to login if they are not logged in.

Reply:
Alright, I just implemented that